pyodide-build/pyodide_build/buildpkg.py

```diff
@@ -19,9 +19,11 @@
 import fnmatch
 from contextlib import contextmanager
 
+from . import pywasmcross
+
 
 @contextmanager
-def chdir(new_dir):
+def chdir(new_dir: "os.PathLike[str]"):
```

Comment:
What would be the difference between os.PathLike[str] and pathlib.Path? It's an abstract class and therefore more general?  Also I think it should work without the quotes?

Reply:
The point is that it should accept either a string or `pathlib.Path`. Without the quotes I think it passed mypy but failed when run normally.

Reply:
It does look like we only ever apply it to `pathlib.Path` though so maybe using that type would be better.